### ISSUE TITLE
setDebug was not found in @nativescript-community/perms

### ISSUE THREAD

**Issue by sivo1981** [farfromrefug/nativescript-gps#6]
@farfromrefug It's not a big issue but can you fix wrong import.

```
WARNING in ../node_modules/@nativescript-community/gps/gps.common.js 7:2-10
"export 'setDebug' was not found in '@nativescript-community/perms'
 @ ../node_modules/@nativescript-community/gps/gps.js
```

I just love how this plugin works specially with GPS provider comparing to [https://github.com/NativeScript/nativescript-geolocation](url)

Love your work.

**Comment by farfromrefug** [comment]
@sivo1981 yes sorry it is already fix need to publish ;)
and thanks!

**Comment by sivo1981** [comment]
I can confirm that fix works in version 2.1.5.
